### ISSUE TITLE
Remove Validation Error on Title

### ISSUE THREAD

**Issue by lewismorris** [StackUp-Digital/Sanity-SEO-Plugin#5]
Currently you'll get a validation error if your `Title for SEO & social sharing` is below 15 chars. There is some cases where it's valid to have a title this short, for example:

`{pageTitle} | MyWebsite.co.uk`, if you had a "Contact Us" title this would only be 10 chars but is a valid title for the pages used case.

**Comment by JamieBradders** [comment]
Yo @lewismorris, thanks for raising this. I'm conscious that different SEO teams might have a different opinion about the optimal minimum number of characters for a page title. I have an idea to make this criteria configurable via the plugin config; would that work in your case?

**Comment by lewismorris** [comment]
@JamieBradders yeah that would work for me. I'll still having the recommend length but the ability to modify the validation lenght would be great.
